Cargo.toml

```diff
@@ -60,6 +60,7 @@ pallet-collator-assignment-runtime-api = { path = "pallets/collator-assignment/r
 pallet-configuration = { path = "pallets/configuration", default-features = false }
 pallet-data-preservers = { path = "pallets/data-preservers", default-features = false }
 pallet-data-preservers-runtime-api = { path = "pallets/data-preservers/runtime-api", default-features = false }
+pallet-external-validators = { path = "pallets/external-validators", default-features = false }
```

Comment:
validator-manager can dissappeara then right?

Reply:
Already removed it, it was not a crate but a `mod validator_manager`.

Reply:
fixed